fs/config/ui.go

```diff
@@ -355,6 +358,39 @@ func newSection() {
 	fmt.Println()
 }
 
+// setDefaultEndpointValueForIDriveE2 gets user region endpoint against the Access Key details by calling the API
```

Comment:
Sorry, I meant factor this into `backend/s3/s3.go` - it doesn't belong here as it is is backend specific.

Reply:
Got it.